nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/src/main/java/org/apache/nifi/processors/script/ExecuteScript.java

```diff
@@ -66,8 +85,8 @@
     @Override
     public Set<Relationship> getRelationships() {
         final Set<Relationship> relationships = new HashSet<>();
-        relationships.add(REL_SUCCESS);
-        relationships.add(REL_FAILURE);
+        relationships.add(ScriptUtils.REL_SUCCESS);
```

Comment:
Yes, I only changed them because I kept the class-level versions of them for backwards compatibility